contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,296 @@
+
+In this example, we can see retention settles at ~20% from 'week 4' onwards in most cohorts. 
+
+That's a useful baseline, but what happens if we filter by just people who use Microsoft Windows?
+
+![new user retention windows](../images/blog/retention-vs-churn/new-user-retention-windows-users.png)
+
+It's obvious Windows users retain worse than the baseline, dipping as low as 15% in week 4 onwards in the 'Dec 11' cohort. 
+
+Do macOS users retain better than Windows users? Let's check...
+
+![new user retention mac os](../images/blog/retention-vs-churn/retention-table-macos.png)
+
+They do! While the exact values vary, macOS users consistently retain better than Windows users. 
+
+This prompts several avenues os investigation. Is the Windows Hedgebox experience worse and, if so why? Does Hedgebox solve some fundamental problem for macOS users that makes them retain better? Are Windows users demographically different to macOS users?
+
+This is just one example using demo data in PostHog. You could also filter users by:
+
+- Their age
+- The browser they use
+- The email account they used to sign up 
+- The country or city they live in
+- The number of files they upload
+- The types of files they upload
+- User job titles, company type or myriad other properties
+
+Cohort retention tables are also useful for tracking revenue retention, especially if you charge based on usage. Positive revenue retention (i.e. more than 100%) indicates users increase their spend over time. Needless to say, this is a good thing.
+
+> 😱 **Pedantic Sidebar:** For those thinking "if retention and churn rate are just the inverse of each other, why can't I create a churn rate table instead?" My answer is this... yes, you could, but it would be a dumb thing to do. Why? Because retention tables make more intuitive sense. Be a rebel if you like, but you're dead to me. 
+
+## How to improve retention and reduce churn
+
+"Let's reduce churn" isn't quite akin to "nobody ever got fired for buying IBM", but it's worth remembering why reducing churn is desirable:
+
+1. High churn is a likely sign you don't have product-market fit.
+
+2. High churn leads to increased sales and/or marketing spend to compensate.
+
+3. Reducing churn compounds – i.e. revenue **and** marketing ROI increase.
+
+So, how do you improve retention and reduce churn? The glib answer is "make product better", but we can do a bit better than that:
+
+- **Check for involuntary churn:** Involuntary churn typically means billing problems – e.g. expired cards, out-of-date info etc. In e-commerce, abandoned baskets could also be an involuntary churn problem.
+
+- **Improve new user activation:** New user activation is an [important B2B product metric](/blog/b2b-saas-product-metrics). Use funnel analysis and session recordings to identify problems in your onboarding flow. Sometimes small tweaks to user flows can have a big impact.
```

Comment:
A failure to activate shouldn't be consider churn IMO (and most people who write about this stuff that I've read take that stance)